tests/utils.py

```diff
@@ -0,0 +1,23 @@
+from wps_tools.testing import client_for
+from pywps import Service
+
+
+def process_err_test(process, params):
+    client = client_for(Service(processes=[process]))
+    # exception_el.text only shows the ProcessError message if loglevel is set to DEBUG
+    if "loglevel=DEBUG" not in params:
+        params += "loglevel=DEBUG"
+    resp = client.get(
+        service="wps",
+        request="Execute",
+        version="1.0.0",
+        identifier=process.identifier,
+        datainputs=params,
+    )
+
+    exception_el = resp.xpath(
+        "/wps:ExecuteResponse/wps:Status/wps:ProcessFailed/"
+        "wps:ExceptionReport/ows:Exception/ows:ExceptionText"
+    )[0]
```

Comment:
Do we need to consider cases where the index is not `0`?

Reply:
I can look into it, I based it off [these tests from pyWPS](https://github.com/geopython/pywps/blob/20e1e254a3f7914e555fa89f363d1f6eb5f3895c/tests/test_exceptions.py#L25)